python/pyspark/mllib/linalg/distributed.py

```diff
@@ -775,6 +775,80 @@ def numCols(self):
         """
         return self._java_matrix_wrapper.call("numCols")
 
+    def add(self, other):
+        """
+        Adds two block matrices together. The matrices must have the
+        same size and matching `rowsPerBlock` and `colsPerBlock` values.
+        If one of the sub matrix blocks that are being added is a
+        SparseMatrix, the resulting sub matrix block will also be a
+        SparseMatrix, even if it is being added to a DenseMatrix. If
+        two dense sub matrix blocks are added, the output block will
+        also be a DenseMatrix.
+
+        :param other: A BlockMatrix
```

Comment:
`param other` and `return` is not necessary here. `A BlockMatrix` doesn't provide much info.